rhel.Dockerfile

```diff
@@ -0,0 +1,129 @@
+#
```

Comment:
Still a mistake to me to have rhel.Dockerfile instead of Dockerfile.rhel. It spread related files to different locations